zproject/db/backends/postgresql/base.py

```diff
@@ -0,0 +1,11 @@
+from django.db.backends.postgresql import base
+
+if False:
+    from typing import Any
+
+class DatabaseWrapper(base.DatabaseWrapper):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None
+        super(DatabaseWrapper, self).__init__(*args, **kwargs)
+        # Always log queries and track times.
+        self.force_debug_cursor = True
```

Comment:
>`# Always log queries and track times.`

I think this comment is a bit misleading. I'll update it. The format hasn't changed. This is the code which is used to capture queries:
```Python
 76     def execute(self, sql, params=None):
 77         start = time()
 78         try:
 79             return super(CursorDebugWrapper, self).execute(sql, params)
 80         finally:                                                                                     
 81             stop = time()
 82             duration = stop - start
 83             sql = self.db.ops.last_executed_query(self.cursor, sql, params)                          
 84             self.db.queries_log.append({                                                             
 85                 'sql': sql,
 86                 'time': "%.3f" % duration,                                                           
 87             })       
 88             logger.debug(
 89                 '(%.3f) %s; args=%s', duration, sql, params,                                         
 90                 extra={'duration': duration, 'sql': sql, 'params': params}                           
 91             )
```
It should work with `write_log_line`.